test/unit/linode_client_test.py

```diff
@@ -476,6 +476,22 @@ def test_join_beta_program(self):
             )
             self.assertEqual(m.call_url, join_beta_url)
 
+    def test_account_transfer(self):
+        """
+        Tests that payments can be retrieved
+        """
+        transfer = self.client.account.transfer()
```

Comment:
Would this function hard to test with integration tests? Other than that tests look good!

Reply:
@ykim-1 Yup! The main issue is that this represents account-wide network transfer so it's difficult to reliably test it across multiple accunts.